astro.config.mjs

```diff
@@ -76,9 +76,22 @@ export default defineConfig({
               label: "Configuration",
               slug: "docs/getting-started/configuration",
             },
+          ],
+        },
+        {
+          label: "Writing First Test",
+          items: [
+            {
+              label: "Appium",
+              slug: "docs/writing-first-test/appium",
+            },
+            {
+              label: "Playwright",
+              slug: "docs/writing-first-test/playwright",
+            },
             {
-              label: "Writing First Test",
-              slug: "docs/getting-started/writing-first-test",
```

Comment:
It'd be good to have [a redirect](https://docs.astro.build/en/guides/routing/#configured-redirects) from the old URL to one of the new ones.